app/src/main/java/org/wikipedia/activity/BaseActivity.kt

```diff
@@ -92,7 +93,8 @@ abstract class BaseActivity : AppCompatActivity() {
         setNavigationBarColor(ResourceUtil.getThemedColor(this, R.attr.paper_color))
         maybeShowLoggedOutInBackgroundDialog()
 
-        ReadingListsSurveyHelper.maybeShowSurvey(this)
+        ReadingListsShareSurveyHelper.maybeShowSurvey(this)
```

Comment:
Somehow the dialog will show twice. 

Not sure if you are able to reproduce it, but the dialog will also show when I open an article. 
<img src="https://user-images.githubusercontent.com/2435576/203394454-5b1bc899-ef29-4f0e-8aa4-87311add282a.jpg" width="480" />

Reply:
Yeah, good observation. The logic should be all fixed now.